### PR TITLE
Selenium: translate deprecated methods

### PR DESCRIPTION
**Pull request by melisabok**
- Update the tests to use the version 4.9.1 of selenium. 
- Using version 4.10 and higher throws an error, I'm waiting for a response here: https://github.com/pytest-dev/pytest-selenium/issues/315

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -106,32 +108,32 @@ def respond_invitation(selenium, request_page, url, accept, quota=None, comment=
...
+            dropdown = selenium.find_element(By.CLASS_NAME, 'dropdown-select__input-container')
             dropdown.click()
             time.sleep(1)
-            values = selenium.find_elements_by_class_name('dropdown-select__option')
+            values = selenium.find_elements(By.CLASS_NAME, 'dropdown-select__option')
             assert len(values) > 0
             values[0].click()
             time.sleep(1)
-            button = selenium.find_element_by_xpath('//button[text()="Submit"]')
+            button = selenium.find_element(By. XPATH, '//button[text()="Submit"]')
```
Comment (xkopenreview) [inline tests/conftest.py:129]: there's some extra space which looks not right

Comment (melisabok) [reply tests/conftest.py]: fixed. It is weird the tests are not failing because of this

File: tests/conftest.py [hunk 2]
```diff
@@ -106,32 +108,32 @@ def respond_invitation(selenium, request_page, url, accept, quota=None, comment=
...
             values[0].click()
             time.sleep(1)
-            button = selenium.find_element_by_xpath('//button[text()="Submit"]')
+            button = selenium.find_element(By.XPATH, '//button[text()="Submit"]')
             button.click()
         elif comment:
             buttons[1].click()
             time.sleep(1)
-            text_area = selenium.find_element_by_css_selector(".note_content_value, [class*='TextareaWidget_textarea']")
+            text_area = selenium.find_element(By. CSS_SELECTOR, ".note_content_value, [class*='TextareaWidget_textarea']")
```
Comment (celestemartinez) [inline tests/conftest.py:134]: extra space here too

File: tests/test_double_blind_conference.py [hunk 3]
```diff
@@ -1539,21 +1540,21 @@ def test_reassign_reviewers_with_conflict_of_interest(self, client, test_client,
...
+        assign_Reviewer_Textbox=selenium.find_element(By.XPATH, '//*[@id="1-add-reviewer"]/div/input')
         selenium.execute_script("arguments[0].click()",assign_Reviewer_Textbox)
 
         #get the dropdown div and check how many dropdown options in it
         dropdown_Options_Present=EC.presence_of_element_located((By.XPATH,'//*[@id="1-add-reviewer"]/div/div'))
         WebDriverWait(selenium,5).until(dropdown_Options_Present)
-        dropdown_Options=selenium.find_element_by_xpath('//*[@id="1-add-reviewer"]/div/div')
-        assert len(dropdown_Options.find_elements_by_xpath('//*[@id="1-add-reviewer"]/div/div/div'))==1
+        dropdown_Options=selenium.find_element(By.XPATH, '//*[@id="1-add-reviewer"]/div/div')
+        assert len(dropdown_Options.find_elements(By. XPATH, '//*[@id="1-add-reviewer"]/div/div/div'))==1
```
Comment (celestemartinez) [inline tests/test_double_blind_conference.py:1557]: extra space here too

File: tests/conftest.py (hunk 2 above)
Comment (melisabok) [reply tests/conftest.py]: fixed

File: tests/test_double_blind_conference.py (hunk 3 above)
Comment (melisabok) [reply tests/test_double_blind_conference.py]: fixed

**Pull request merged**